test/test.lua

```diff
@@ -1268,6 +1309,179 @@ function test.reset_device()
    tester:assertTensorEq(tf, u:float(), 1e-6, "values not equal after restoring the RNG state")
 end
 
+function test.maskedSelect()
+   local n_row = math.random(minsize,maxsize)
+   local n_col = math.random(minsize,maxsize)
+
+   -- contiguous, no result tensor, cuda mask
+   local x = torch.randn(n_row, n_col):float()
+   local mask = torch.DoubleTensor():rand(n_row*n_col):mul(2):floor():byte():resize(n_row,n_col)
```

Comment:
Nit: This can be shortened to
`torch.DoubleTensor(n_row, n_col):uniform():round():byte()`